core/src/main/scala/stryker4s/extension/mutationtype/ConditionalExpression.scala

```diff
@@ -0,0 +1,20 @@
+package stryker4s.extension.mutationtype
+
+import scala.meta._
+
+case object If {
+  def unapply(ifStatement: Term.If): Option[Term] = Some(ifStatement.cond).filterNot(_.is[Lit.Boolean])
+}
+case object While {
+  def unapply(whileStatement: Term.While): Option[Term] = Some(whileStatement.expr).filterNot(_.is[Lit.Boolean])
+}
+case object Do {
```

Comment:
Could you rename this to DoWhile? Just 'Do' is not very clear

Reply:
Fair enough, I only called it Do because that's what the Term is called in scala-meta.